.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [Windows, Ubuntu, MacOS]
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Could we please hold off on this until we have one release with a CLI? I know it's a big ask, and that you are tired of dealing with Python 2, but I can step up and help deal with that if you want.

Reply:
I empathise with your situation but, I have to say no.

As I said in https://github.com/pradyunsg/installer/issues/42, I'm not interested in spending my free time for supporting anyone still stuck on Python 2. Python 2 support is singlehandedly the reason I haven't been motivated to work on this project in months.

Quoting from https://curiousefficiency.org/posts/2015/04/stop-supporting-python26.html:

> upstream open source developers shouldn't feel obliged to donate their own time to help end users cope with organisations that aren't yet able to upgrade their internal infrastructure in a more timely fashion.

Right now, it's 635 days since the upstream EoL of Python 2. That's over 20 months, and that's after the support was extended for an 5 extra years by the CPython devs, beyond the initial fairly-long support period.

From my perspective, the CLI can be built externally _right now_ if someone wants. A Python 3 script can be used to install Python 2 wheels. Someone can even maintain a Python 2 fork where they backport the changes improvements and bugfixes in this project. Overall, if a person really wants Python 2 support still, it's gotta worthwhile for them to do additional effort -- that's the cost of EoL software -- and I genuinely hope that person isn't donating their limited free time to make migrations cheaper for large organisations.

Reply:
Fair enough. I will provide this as an external package.

> and I genuinely hope that person isn't donating their limited free time to make migrations cheaper for large organisations

My interest is to make the migration possible for Arch Linux, which is an organization entirely run by volunteers like me, so :worried: